### DISCUSSION TITLE
Recurring payment: possible to add it before date?

### DISCUSSION THREAD

**Discussion by macuros** [firefly-iii/firefly-iii#5266] [Firefly III] [answered]
Hello,

First of all, I really like FireFly! Keep up!

As I'm fairly new to this, is there any way to add a recurring payment before the date that is set? Sometimes my bills comes earlier and I'd like to add that payment before.

**Comment by JC5** [comment, marked as answer]
It's not possible yet, but it's on my list to add in the new layout!

**Comment by macuros** [comment]
Perfect! Thank you.
